manifests/b/Balsamiq/Wireframes/4.7.4/Balsamiq.Wireframes.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Balsamiq.Wireframes
+PackageVersion: 4.7.4
+InstallerType: exe
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://builds.balsamiq.com/bwd/Balsamiq_Wireframes_4.7.4_x64_Setup.exe
+  InstallerSha256: 43926B24FF2EF04EE260B3AB9FCEF7DF20CFD02682F8ED704272880474D2DF1C
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://builds.balsamiq.com/bwd/Balsamiq_Wireframes_4.7.4_x64_Setup.exe
+  InstallerSha256: 43926B24FF2EF04EE260B3AB9FCEF7DF20CFD02682F8ED704272880474D2DF1C

```

```suggestion
  InstallerSha256: 43926B24FF2EF04EE260B3AB9FCEF7DF20CFD02682F8ED704272880474D2DF1C
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://builds.balsamiq.com/bwd/Balsamiq_Wireframes_4.7.4_x64_Setup.exe
  InstallerSha256: 43926B24FF2EF04EE260B3AB9FCEF7DF20CFD02682F8ED704272880474D2DF1C
  InstallerSwitches:
    Custom: /ALLUSERS
```
Copied from a Linwood manifest.